modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/alert-info/index.css

```diff
@@ -0,0 +1,8 @@
+.alert-info {
+	bottom: 70px;
+	box-shadow: 0 0.5rem 2rem -0.25rem rgba(0, 0, 0, 0.3);
+	left: 20px;
+	position: fixed;
+	width: 21rem;
+	z-index: 5000;
+}
```

Comment:
This class belongs to bootstrap too, the idea is override bootstrap?

Could we use a different class for this purpose?

Reply:
Replace some for bootstrap, but I needed to leave some 2 properties still.